src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMapping.java

```diff
@@ -106,18 +105,17 @@ public static boolean isInterpreterRequiredHearingOptions(HearingOptions hearing
 
     public static List<CaseCategory> buildCaseCategories(SscsCaseData caseData) {
         // TODO Adjournment - Check this is the correct logic for Adjournment
-        // TODO SSCS-10116 - Replace SessionCaseCodeMapping with commons version
         List<CaseCategory> categories = new ArrayList<>();
 
-        SessionCaseCodeMapping sessionCaseCode = getSessionCaseCode(caseData);
+        SessionCategoryMap sessionCaseCode = getSessionCaseCode(caseData);
```

Comment:
In the future could you please qualify static calls like this to make it clear where the method is?